base/range.jl

```diff
@@ -41,6 +41,9 @@ equal to 1, which produces:
 `a:s:b` is similar but uses a step size of `s` (a [`StepRange`](@ref) or
 [`StepRangeLen`](@ref)). See also [`range`](@ref) for more control.
 
+To create a descending range, use `reverse(a:b)` or a negative step size, e.g. `b:-1:a`.
+Otherwise when `b < a`, an empty range will be constructed and normalized to `a:a-1`.
```

Comment:
```suggestion
Otherwise when, `b < a`, an empty range will be constructed and normalized to `a:a-1`.
```

Reply:
Normal English punctuation rules require a comma after "otherwise".  It doesn't matter that there is an additional comma-separated clause after that word.